spec/features/business/slug_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+describe "business show page", type: :feature do
+  before :each do
+    @business = FactoryGirl.create(:business)
```

Comment:
Okay. I personally like to know where it's coming from. But will change it in the future.